src/content/community.md

```diff
@@ -52,6 +53,39 @@ There are several ways to stay up-to-date with Firefox add-on technology and con
 
 Stay up to date with the latest information about add-on technology on the [Add-ons blog](https://blog.mozilla.org/addons).
 
+### Office hours <span id="office-hours"></span>
+
+The Firefox Add-ons team holds one-on-one office hours sessions to help developers make their extensions available on Android. Developers can also use the [Firefox for Android Add-ons](https://discourse.mozilla.org/c/add-ons/android/9393) forum to discuss issues asynchronously.
+
+<!-- Google Calendar Appointment Scheduling begin -->
+<link href="https://calendar.google.com/calendar/scheduling-button-script.css" rel="stylesheet">
+<script src="https://calendar.google.com/calendar/scheduling-button-script.js" async></script>
```

Comment:
I don't think this will work once deployed because `https://calendar.google.com/calendar/` isn't specified in the CSP.

Reply:
Good call. I'll revise the PR with this and [Rob's comment](https://github.com/mozilla/extension-workshop/pull/1698#issuecomment-1726184721) in mind.